frontend/src/v5/ui/components/viewer/drawingViewer/drawingViewer.component.tsx

```diff
@@ -0,0 +1,113 @@
+/**
+ *  Copyright (C) 2024 3D Repo Ltd
```

Comment:
The toolbar seems to behave differently with svgs and pngs:
- with SVGs, as the mouse hovers the toolbar, events seem to not propagate. For instance, hover over the toolbar and move the wheel up and down. nothing happens;
- with PNGs, the same will produce a zooming.

(Note, also the bug with dragging the zooming icon only affect PNGs)

Reply:
This is because atm the svg is in a intermediate container. Wont address that  for now because the structure will probably change when addressing the performance issues.